### PR TITLE
Add Swift 3

### PR DESCRIPTION
**Pull request by diegoserranoa**

### CODE REVIEW

File: source/ios.html.erb [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: iOS Documentation
 language_tabs:
   - objective_c: Objective-C
   - swift: Swift
```
Comment (alfonsocora) [inline source/ios.html.erb]: should we rename this to "Swift 2"?

File: lib/swift_lexer.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require 'rouge'
+module Rouge
+  module Lexers
+    class SwiftThree < Rouge::Lexers::Java
```
Comment (alfonsocora) [inline lib/swift_lexer.rb]: didn't know they were so similar

File: source/ios.html.erb (hunk 1 above)
Comment (diegoserranoa) [inline source/ios.html.erb]: True that

**Comment by alfonsocora** [comment]
@diegoserranoa great! :shipit:

**Pull request merged**